apps/server/src/treematch/treematch.controller.ts

```
@@ -0,0 +1,83 @@
+    );
+  }
+
+  // :contributionId is TTC's ProjectContribution id, the only id the web has.
+  @Patch('/projects/:id/contributions/:contributionId/ignore')
+  @ProjectRoles('owner', 'admin')
+  @UseGuards(ProjectPermissionsGuard)
+  async setContributionIgnore(
+    @Param('contributionId', ParseIntPipe) contributionId: number,
+    @Body() dto: SetContributionIgnoreDto,
+  ) {
+    return this.treeMatchService.setContributionIgnore(contributionId, dto);
```

ESLint

[error] Delete `;`

(prettier/prettier)